docs_src/getting-started/Ch-GettingStartedSnapUsers.md

```diff
@@ -0,0 +1,127 @@
+
+!!! Warning
+    For full security confinement, the snap should be installed on an Ubuntu 16.04 LTS or later Desktop or Server, or a system running Ubuntu Core 16 or later.
+
+## Installing EdgeX Foundry as a snap
+
+The snap is published in the snap store at https://snapcraft.io/edgexfoundry. You can see the current revisions available for your machine's architecture by running the command:
+
+```
+$ snap info edgexfoundry
+```
+The snap can be installed using snap install. To install the snap from the edge channel:
+
+```
+$ sudo snap install edgexfoundry --edge
+```
+You can install a specific release using the --channel option. For example to install the Fuji release of the snap:
+
+```
+$ sudo snap install edgexfoundry --channel=fuji
+```
+Lastly, on a system supporting it, the snap may be installed using GNOME (or Ubuntu) Software Center by searching for edgexfoundry.
+
+!!! Note
+    The snap has only been tested on Ubuntu Desktop/Server versions 18.04 and 16.04, as well as Ubuntu Core versions 16 and 18.
+
+!!! Warning
+    Don't install the EdgeX snap on a system which is already running one of the included services (e.g. Consul, Redis, Vault, ...).
+
+## Using the EdgeX snap
+
+Upon installation, the following EdgeX services are automatically and immediately started:
+
+    - consul
+    - redis
+    - core-data
+    - core-command
+    - core-metadata
+    - security-services (see note below)
+
+The following services are disabled by default:
+
+    - app-service-configurable (required for Kuiper and support-rulesengine)
+    - device-virtual
+    - kuiper
+    - support-logging
+    - support-notifications
+    - support-rulesengine (deprecated)
+    - support-scheduler
+    - sys-mgmt-agent
+
+Any disabled services can be enabled and started up using snap set:
+
+```
+$ sudo snap set edgexfoundry support-notifications=on
+```
+To turn a service off (thereby disabling and immediately stopping it) set the service to off:
+
+```
+$ sudo snap set edgexfoundry support-notifications=off
+```
+
+All services which are installed on the system as systemd units, which if enabled will automatically start running when the system boots or reboots.
+
+## Configuring individual services
+
+All default configuration files are shipped with the snap inside $SNAP/config, however because $SNAP isn't writable, all of the config files are copied during snap installation (specifically during the install hook, see snap/hooks/install in this repository) to $SNAP_DATA/config.
+
+!!! Note
+    As the core-config-seed was removed as part of the Geneva release, services self-seed their configuration on startup. This means that if a service is started by default in the snap, the only way to change configuration is to use the Consul UI or kv REST API. Services that aren't started by default (see above) will pickup any changes made to their config files when enabled.
+
+```
+$ sudo snap restart edgexfoundry
```

Comment:
This command sees out of place -- nothing talks about it

Reply:
taking out per conversation with Tony